### ISSUE TITLE
copy/move on open folders

### ISSUE THREAD

**Issue by sbraaa** [linuxmint/nemo#2944]
I would like that right click menu option "copy to" includes list of open folders,
anyone know if it is possible and how to enable it? 
I try to find some extensions but it seems them doesn't exists
Thanks

**Comment by Jeremy7701** [comment]
Have you checked preferences -> context menu, copy-to?
This allows you to copy to any bookmarked location with a right-click.

Producing a list of all "open" folders would be a bit more difficult. ;-)

**Comment by sbraaa** [comment]
> Have you checked preferences -> context menu, copy-to?
> This allows you to copy to any bookmarked location with a right-click.
> 
> Producing a list of all "open" folders would be a bit more difficult. ;-)

Yes, thanks I know very well but you understand exactly what I mean: it would be very tedious add a bookmark to use it only one time.
It would be more useful have the list of all open folders.
From your answer I would argue that this feature doesn't exist :-(

**Comment by Jeremy7701** [comment]
Indeed.
I assume a twin pane window via F3 or a tabbed window wouldn't be suitable?
The twin pane option includes a "copy to other pane" in the right-click copy list.
But the more general feature doesn't exist.
